tests/load-tests/ci-scripts/utility_scripts/playwright-update-tokens.py

```diff
@@ -0,0 +1,116 @@
+#
+# Setup:
+#     python -m venv venv
+#     source venv/bin/activate
+#     pip install playwright
+#     playwright install   # download Chromium browser
+#
+# Running:
+#     Consider PLAYWRIGHT_HEADLESS = True so it runs in background
+#
+#     ci-scripts/utility_scripts/playwright-update-tokens.py
+#
+#     To sort final file I have used:
+#     cat users-new.json | jq '. | sort_by(.username[11:] | tonumber)'
+#
+#     Note I was not able to run with more threads as I was getting
+#     "Access Denied" errors. I guess it was some rate limiting.
+
+import playwright.sync_api
+import os
+import time
+import json
+import concurrent.futures
+
+PLAYWRIGHT_HEADLESS = False
+PLAYWRIGHT_VIDEO_DIR = "videos/"
+
+def thread(user):
+    username = user["username"].replace("-", "_")
+    password = user["password"]
+
+    with playwright.sync_api.sync_playwright() as p:
+        browser = p.chromium.launch(
+            headless=PLAYWRIGHT_HEADLESS,
+        )
+        context = browser.new_context(
+            record_video_dir=PLAYWRIGHT_VIDEO_DIR,
+        )
+        page = context.new_page()
+
+        page.goto("https://console.dev.redhat.com")
+        page.wait_for_url("https://sso.redhat.com/**")
+
+        # Accept cookies
+        cookies_iframe = page.frame_locator('iframe[name="trustarc_cm"]')
+        cookies_button = cookies_iframe.get_by_role("button", name="Agree and proceed with standard settings")
+        cookies_button.click()
+
+        # Wait for login form and use it
+        page.wait_for_selector('//h1[text()="Log in to your Red Hat account"]')
+        input_user = page.locator('//input[@id="username-verification"]')
+        time.sleep(1)
+        input_user.fill(username)
+        button_next = page.locator('//button[@id="login-show-step2"]')
+        button_next.click()
+        button_next.wait_for(state="hidden")
+        input_pass = page.locator('//input[@id="password"]')
+        input_pass.wait_for(state="visible")
+        input_pass.fill(password)
+        page.locator('//button[@id="rh-password-verification-submit-button"]').click()
+
+        # Wait for console to load and go to OpenShift Token page
+        page.wait_for_url("https://console.dev.redhat.com/**")
+        page.wait_for_selector('//h2[text()="Welcome to your Hybrid Cloud Console."]')
+        page.goto("https://console.dev.redhat.com/openshift/token")
+        page.wait_for_url("https://console.dev.redhat.com/openshift/token**")
+        page.wait_for_selector('//h2[text()="Connect with offline tokens"]')
+
+        # Wait for token
+        button_token = page.locator('//button[text()="Load token"]')
+        if button_token.is_visible():
+            button_token.click()
+        while True:
+            input_token = page.locator('//input[@aria-label="Copyable token" and not(contains(@value, "ocm login "))]')
+            input_token_value = input_token.get_attribute("value")
+            if len(input_token_value) > 10:
```

Comment:
why are we only accepting if the length is more than 10? Wdyt about adding a comment for this?

Reply:
I do not remember exactly, but I think that value is populated asynchronously and from the start it is set to `null` or empty string or something like that. I did not wanted to rely on one specific value in case somebody decides to change `null` to `Loading` or so, so used value bigger than 0 and smaller than normal token length that is a nice number :-)

Reply:
Added comment

Reply:
Thank you!